view/frontend/templates/js/replacejs.phtml

```diff
@@ -1226,6 +1228,9 @@ $isLoadTrackJsDynamic = $block->isDisableTrackJsOnNonBoltPages() && $isLoadConne
             onElementReady(bolt_button_selector, function(element) {
                 checkout_buttons = true;
                 processButtons();
+                if (boltConfig.always_present_checkout) {
+                    callConfigureWithPromises();
```

Comment:
I am not sure this work. We call `callConfigureWithPromises()` when we know that next moment we receive new values from the server, to prevent checkout with outdated cart we have this moment.
It doesn't look like it's always the case for the always present checkout button.
But if we do not receive new data this call breaks checkout.

The simplest solution here is prohibit minicart button when we have popup button as we discussed in asana. Let's close this PR.